es/utils/aepp-wallet-communication/wallet-detector.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-undef */
+/*
+ * ISC License (ISC)
+ * Copyright (c) 2018 aeternity developers
+ *
+ *  Permission to use, copy, modify, and/or distribute this software for any
+ *  purpose with or without fee is hereby granted, provided that the above
+ *  copyright notice and this permission notice appear in all copies.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+ *  REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+ *  AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+ *  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ *  LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+ *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ *  PERFORMANCE OF THIS SOFTWARE.
+ */
+
+/**
+ * Browser runtime connector module
+ *
+ * This is the complement to {@link module:@aeternity/aepp-sdk/es/utils/wallet-connection}.
+ * @module @aeternity/aepp-sdk/es/utils/wallet-connection/browser-runtime
+ * @export BrowserRuntimeConnection
+ * @example import BrowserRuntimeConnection from '@aeternity/aepp-sdk/es/utils/wallet-connection/browser-runtime'
+ */
+import AsyncInit from '../async-init'
+import BrowserRuntimeConnection from './wallet-connection/browser-runtime'
+import { METHODS } from './schema'
+
+const wallets = {}
+
+const handleDetection = (onDetected) => (msg) => {
+  if (!msg || !msg.data || !msg.data.method || msg.data.type === 'webpackOk') return undefined
```

Comment:
here we could probably have our own `msg.data.type` ...so we only check if `msg.data.type == 'ourDataType'`...?